src/js/components/PodDetail.js

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
+import React from 'react';
+
+import Breadcrumbs from './Breadcrumbs';
+import Pod from '../structs/Pod';
+import PodHeader from './PodHeader';
+import TabsMixin from '../mixins/TabsMixin';
+
+const METHODS_TO_BIND = [
+];
+
+class PodDetail extends mixin(TabsMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.tabs_tabs = {
+      instances: 'Instances',
+      configuration: 'Configuration',
+      debug: 'Debug'
+    };
+
+    this.state = {
+      currentTab: Object.keys(this.tabs_tabs).shift()
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  renderConfigurationTabView() {
```

Comment:
I understand your concerns, but we shouldn't extend the scope due the named reasons and  the time constrains. Let's create an issue and add the routes once we've merged the pods stuff into master.